app/full_text_search.py

```diff
@@ -42,25 +73,40 @@ def search(repo, query_txt, max_results):
     results = []
     if not query_txt:
         return results
-    index = appengine_search.Index(name=PERSON_FULL_TEXT_INDEX_NAME)
-    query_txt += ' AND (repo: ' + repo + ')'
+
+    # escape double quote for enclose each query_txt
```

Comment:
It is not "escape". Escaping is something like:

    " -> \"

You should say "remove double quotes" instead. The whole sentence can be e.g.,
"Remove double quotes so that we can safely apply enclose_in_double_quotes()."